Description: Patch to check if test suite failures are prompt variables related
Author: Peter Stephenson <p.stephenson@samsung.com>
Bug: http://www.zsh.org/cgi-bin/mla/redirect?WORKERNUMBER=35042
Origin: http://www.zsh.org/cgi-bin/mla/redirect?WORKERNUMBER=35044

Index: zsh/Test/A02alias.ztst
===================================================================
--- zsh.orig/Test/A02alias.ztst	2015-05-06 10:17:19.000000000 +0200
+++ zsh/Test/A02alias.ztst	2015-05-06 17:48:40.000000000 +0200
@@ -59,7 +59,7 @@
 >And aliases are expanded
 
   $ZTST_testdir/../Src/zsh -fis <<<'
-  PROMPT=""
+  PROMPT="" PS2="" PS3="" PS4="" RPS1="" RPS2=""
   exec 2>&1
   alias \{=echo
   { begin
@@ -72,7 +72,7 @@
 *>*5*{end
 
   $ZTST_testdir/../Src/zsh -fis <<<'
-  PROMPT=""
+  PROMPT="" PS2="" PS3="" PS4="" RPS1="" RPS2=""
   exec 2>&1
   alias -g S=\"
   echo S a string S "
